backend/logs_helper/log_service.py

```diff
@@ -0,0 +1,17 @@
+from django_redis import get_redis_connection
+
+
+class LogService:
+    @staticmethod
+    def remove_logs_on_logout(session_id):
+
+        if session_id:
+            # Get the Redis connection
+            r = get_redis_connection("default")
+
+            key_pattern = f"logs:{session_id}*"
+
+            # Retrieve keys matching the pattern and delete them
+            keys = r.keys(key_pattern)
```

Comment:
Fetching all the matching keys together like this will be a problem when we have large number of keys could consume more memory, assuming that can happen in this case of storing logs and notification ? @jagadeeswaran-zipstack @tahierhussain 

Alternative: Use `scan_iter()`
 https://stackoverflow.com/questions/22255589/get-all-keys-in-redis-database-with-python

Please take a look and make a call accordingly. Maybe @hari-kuriakose or @ritwik-g can help here take a call.

Reply:
We plan to implement pagination in the next iteration. So we won't be fetching all of the keys.
cc: @tahierhussain

Reply:
@jagadeeswaran-zipstack @tahierhussain If this won't be picked up soon or moved as a tech-debt the performance issue will remain till then, i believe. Anyways please take a call accordingly.

Reply:
@athul-rs I think we could ignore the memory implication for now. Looks like we are only fetching the keys related to a particular session. That too we are not loading the value in the keys.

I guess we are good here.

Reply:
Resolving. @jagadeeswaran-zipstack